### DISCUSSION TITLE
Where do you find up to date documentation about GDM? I don't seem to find quality sources

### DISCUSSION THREAD

**Discussion by thepragmaticmero** [realmazharhussain/gdm-settings#255] [Q&A]
I don't seem to find quality sources, the only helpful information are just forum posts that are really scattered. This [Gnome page](https://help.gnome.org/admin/gdm/stable/configuration.html.en) doesn't have the `WaylandEnable=false` parameter written on it, Gnome documentation in general seems out of date. Where do you find it, on their gitlab or something?

**Comment by realmazharhussain** [comment]
Information about customizing GDM can be found on Wikis of distros, e.g. [Arch Wiki](https://wiki.archlinux.org/title/GDM).

**Reply by realmazharhussain** [reply]
I've heard Gentoo Wiki is good too.

**Comment by thepragmaticmero** [comment]
Thanks :D, it really seems weird to me that Gnome can't do First hand documentation properly.
